src/helpers/agGrid.ts

```diff
@@ -155,7 +139,7 @@ export const getColumnHeaders = (
             // attribute uri or identifier
             // attribute value uri
             // measure uri or identifier
-        const drillItems: IDrillItem[] = [...parentDrillItems];
+        const drillItems: IMappingHeader[] = [...parentDrillItems];
```

Comment:
use clone object instead. this is caused by mutating drillItems inside `assignDrillItemsAndType` method 😠